telegram/_bot.py

```diff
@@ -310,6 +312,16 @@ def __init__(
         self._freeze()
 
     async def __aenter__(self: BT) -> BT:
+        """
+        |async_context_manager| :meth:`initializes <initialize>` the Bot.
+
+        Returns:
+            The initialized Bot instance.
+
+        Raises:
+            :exc:`Exception`: If an exception is raised during initialization, :meth:`shutdown`
+            is called in this case.
```

Comment:
```suggestion
                is called in this case.
```
need to indent the next line to fix doc build. Apply this for all the `__aenter__`'s